ui/src/track-layout/track-layout-search-api.ts

```diff
@@ -0,0 +1,27 @@
+import {
```

Comment:
Jälleen uusi tiedosto, ok vai laitetaanko sisältö muualle?

Reply:
Tämä lienee ihan OK paikka (ainakin näin off the top of my head.) Siirretään sitten muualle myöhemmin jos keksitään joku parempi joskus.